src/game/Pet.cpp

```diff
@@ -596,7 +595,7 @@ void Pet::Update(uint32 update_diff, uint32 diff)
 
 void Pet::RegenerateAll(uint32 update_diff)
 {
-    // regenerate focus
+    // regenerate power
```

Comment:
This right here is a mistake... doing a whole new commit to reverse it I will not. So I am waiting for reviewing / testing results and correcting it with the things I have to correct from feedback anyway, cause... of course there is stuff I have missed or could have optimized better. But this should say "regenerate focus and health"... Honestly just writing this comment to make sure I don't forget.